test/Test/src/Time/Spec.enso

```diff
@@ -0,0 +1,18 @@
+from Base import all
+import Base.Test
+import Base.Time
+
+import Test.Time.Timer.Spec as Timer_Spec
```

Comment:
I decided to change this structure, as the depth is silly. I think you should just have the directory `Time` and in it things like `Timer_Spec.enso`, `Date_Spec.enso`, etc.